src/main/index.ts

```diff
@@ -65,14 +65,23 @@ log.transports.file.format = "[{y}-{m}-{d} {h}:{i}:{s}.{ms}][{processType}][{lev
 log.eventLogger.format = "Electron event {eventSource}#{eventName} observed";
 
 log.hooks.push((message, transport) => {
-  if (transport !== log.transports.file) return message;
-
-  // Annoyingly spammed log from YTM that doesn't need to get added to the log file
-  if (message.data[0].includes("Third-party cookie will be blocked.")) return false;
-  for (let i = 0; i < message.data.length; i++) {
-    let dataString: string = message.data[i] as string;
-    dataString = dataString.replaceAll(/(?<=((https|http):\/\/)?.{1,64}(\..{1,64})?\..{1,64}\/)([\S]+)/gm, "[REDACTED]");
-    message.data[i] = dataString;
+  if (transport !== log.transports.file) { return message; }
+
+  const isSpamMessage = (data: any) => {
+    return typeof data === 'string' && data.includes("Third-party cookie will be blocked.");
+  };
+
+  if (message?.data?.[0] && isSpamMessage(message.data[0])) {
+    return false;
+  }
```

Comment:
I feel like the `isSpamMessage` function migth be better moved to a different place, and having the


```js
if (transport !== log.transports.file) return message;
if (message?.data?.[0] && isSpamMessage(message.data[0])) return false;
````
next to it to be the clear "Early escapes" from the function grouped together.